components/form-control/script.js

```diff
@@ -58,6 +74,7 @@ class GamefaceFormControl extends HTMLElement {
         this.traverseFormElements(this, (element) => {
             //We dont need data from invalid elements and elements that are having type "submit"
             if (!this.isValidFromElement(element) || this.isElementUsedToSubmit(element)) return;
+            if (!this.toCustomElement(element).willSerialize()) return;
```

Comment:
`getFormSerializedData(formElements)`

```suggestion
        for (const formElement of formElements) {
           if (!this.toCustomElement(formElement).willSerialize()) continue;
           this.serializeElementData(formElement, params);
        }
```